src/SdkCommon/ClientRuntime.Azure.Authentication/ActiveDirectoryServiceSettings.cs

```diff
@@ -28,6 +28,20 @@ public sealed class ActiveDirectoryServiceSettings
             ValidateAuthority = true
         };
 
+        private static readonly ActiveDirectoryServiceSettings AzureUSGovernmentSettings = new ActiveDirectoryServiceSettings
```

Comment:
@horrion Apart from the endpoints (and thank you for adding the ARM endpoint), I am more interested to know if these endpoints do work for authenticating against various cloud environments? And hence wanted to know if you have used the piece of code that you have contributed?
Or was your intention to simply add those endpoints?
Because majority of our Authentication scenarios are via our TestFramework when automating our tests
https://github.com/Azure/azure-sdk-for-net/blob/vs17Dev/src/SdkCommon/TestFramework/ClientRuntime.Azure.TestFramework/TestEnvironment.cs#L254